### DISCUSSION TITLE
Private tap formula can be installed but failed on action of broken dependencies

### DISCUSSION THREAD

**Discussion by ezchi** [Homebrew/discussions#1809] [Everyday usage] [answered]
I have my private formula ezchi/missing piece/SystemC

It can be installed with `brew install ezchi/missing piece/systemc` but failed on GitHub action as

```
==> brew install --only-dependencies ./systemc--2.3.3.catalina.bottle.1.tar.gz
==> brew install ./systemc--2.3.3.catalina.bottle.1.tar.gz
==> brew linkage --test ezchi/missingpiece/systemc
==> FAILED
Broken dependencies:
  /usr/local/opt/llvm/lib/libc++.1.dylib (llvm)

==> brew install --only-dependencies --include-test ezchi/missingpiece/systemc
Error: linkage failed
==> FAILED ezchi/missingpiece/systemc
Error: 1 failed step!
brew linkage --test ezchi/missingpiece/systemc
Error: Process completed with exit code 1.
```

**Comment by SMillerDev** [comment]
I'm not sure what to do with this information. Thanks for the information I guess?

**Comment by gromgit** [comment]
What's the output of `brew info ezchi/missingpiece/systemc`?

**Comment by carlocab** [comment, marked as answer]
You have linkage with `llvm`'s `libc++`, so `llvm` needs to be a runtime dependency, not a build-only one.

Alternatively, you can try to make it link with the system `libc++` by removing `llvm`'s `opt_lib` from `HOMEBREW_LIBRARY_PATHS`. See `deno`, `v8`, or `apache-arrow` for examples. If this is successful you should be able to keep `llvm` as a build dependency, but it doesn't always work. It will probably depend on how the build system handles `CLANG_BASE_PATH`.
